wordpress/wp-elsa/wp-elsa.php

```diff
@@ -0,0 +1,118 @@
+<?php
```

Comment:
So are Cyber Command / DoITT ok with us installing this custom code on our wpengine instance? What is the approval process? Do we need to go through an approval process every time we change the code?

If I wanted to add my own custom php function (say, manually building my own simple "webhook" by hitting a hard coded endpoint every time a wordpress post is edited), would that be ok?

Reply:
Yes, according to @jisaf the process of having this go through Cyber would be more streamlined compared to if we are to use plugins that are not within WPEngine because it's worked by an unapproved third party